environment.py

```diff
@@ -75,13 +75,13 @@ class EnvAssigner(Visitor):
 
     """
 
-    def __init__(self, parent_env: Environment):
+    def __init__(self, parent_env: Environment) -> None:
```

Comment:
I believe the `__init__(…) -> None` has been fixed in MyPy, since my `--strict` doesn't report errors for these. What version of MyPy are you using? For me:

```shell
$ mypy --version
mypy 0.641
```

Reply:
Ah, I see. *technically* it's now only needed for `__init__` if no arguments are typed (so it knows to type-check them). I can see wanting `-> None` everywhere for consistency, so I retract that request.

Reply:
I actually didn't know they had changed the behavior of mypy. Looks like that change happened in the last month or so. I actually prefer leaving out the `-> None` on init if it has typed parameters. `-> None` is kind of gross imo.